ClassTypes/PowerShell/EthicalHacking/README.md

```diff
@@ -1,40 +1,71 @@
 # Introduction
-These scripts will guide you to create and setup an Azure Lab Services lab account that is configured to run an [ethical hacking class](https://docs.microsoft.com/en-us/azure/lab-services/classroom-labs/class-type-ethical-hacking). Part 1 of these instructions will be to create the lab account on Azure. Part 2 of these instructions will be to prepare the template VM instance of the newly created lab to be used by your class.
+
+These scripts will guide you to create and setup an Azure Lab Services lab that is configured to run an [ethical hacking class](https://docs.microsoft.com/azure/lab-services/classroom-labs/class-type-ethical-hacking). Part 1 of these instructions will be to create the lab plan on Azure. Part 2 of these instructions will be to prepare the template VM instance of the newly created lab to be used by your class.
 
 - - - -
 
-## Part 1 - Create a Lab Account 
-This script will help create a Lab Account on your Azure account from your local machine.
+## Part 1 - Create a lab plan
```

Comment:
"Lab plan and lab"?  Or you could just say, "Create Azure-based Lab Resources" or something...